### PR TITLE
Problem solution for cron job error

### PR DESCRIPTION
**Pull request by pierrejochem**
Solves a problem with a cron job: https://github.com/owncloud/search_lucene/issues/106

### CODE REVIEW

**Comment by karlitschek** [comment]
@butonic What do you think?

File: db/statusmapper.php [hunk 1]
```diff
@@ -169,15 +169,17 @@ public function getUnindexed() {
 				$this->logger->
 					debug( 'expected string or instance of \OC\Files\Mount\Mount got ' . json_encode($mount) );
 			}
-			//only index local files for now
-			if ($storage->isLocal()) {
```
Comment (butonic) [inline db/statusmapper.php]: how about `if ($storage && $storage->isLocal()) {`? Can you create a PR? Failing tests unrelated. Will merge when you update the PR :)

**Comment by pierrejochem** [comment]
Is changed ;-)

**Comment by butonic** [comment]
good enough, I won't bother you to squash and keep the patch minimal. Thx for diggin into this!

**Pull request merged**